### ISSUE TITLE
ISSN 1461-7404 | Social Compass | Spracherkennung fehlerhaft

### ISSUE THREAD

**Issue by kdr154** [ubtue/DatenProbleme#1198]
https://journals.sagepub.com/toc/scpa/67/1
Die Spracherkennung ist fehlerhaft. In 041a steht immer "eng". Auch ein Überschreiben der inzwischen verbesserten Voreinstellung der harvester.conf mit zotero_expected_languages = eng,fre ändert daran nichts.

Bsp.:

![grafik](https://user-images.githubusercontent.com/29707105/96738874-78b8f280-13bf-11eb-93f7-b5244e81c230.png)

**Comment by socheres** [comment]
@kdr154 @IxKo wenn der Translator falsche Sprache liefert, kann sie durch Eingabe der Werte mit **Stern** z.B. `*title:eng,fre` in ZEDER-Spalte "sprz"  korrigiert werden. Testen Sie nochmal auf benu, indem Sie  "in Additional config" folgendes eingeben: `zotero_expected_languages =*title:eng,fre` @mtrojan-ub werden die Werte in "sprz" auch täglich in der Config enhancementmap aktualisiert?

**Comment by kdr154** [comment]
So sieht es recht gut aus. Es ist eine Sache dabei, die auf diesem Weg schwer, kaum, nicht zu lösen sein dürfte. Hier besteht der Titel aus englischen und französischen Bestandteilen: Introduction: ‘Open Secularism’ from the Margins La « laïcité ouverte » vue des marges. Der Text ist eigentlcih auf Englisch, in 041 steht aber fre. Deshalb würde ich dieses Issue als gelöst betrachten.

**Comment by kdr154** [comment]
@socheres hier würde mich interessieren, ob man auch auf mehrer Felder Bezug nehmen kann. Etwa in der Form:
zotero_expected_languages =*title,*abstract_note:eng,fre

**Comment by socheres** [comment]
@kdr154 ja, aber denke so:`*title+*abstract:eng,fre` @mtrojan-ub stimmt die Formel?

**Comment by mtrojan-ub** [comment]
`*` darf nur am Anfang vorkommen, korrekt wäre also: `*title+abstract:eng,fre`
